centreon/www/class/centreonMonitoring.class.php

```diff
@@ -128,53 +128,69 @@ public function getServiceStatusCount($host_name, $objXMLBG, $o, $status, $obj)
      */
```

Comment:
Maybe be more precise in the phpdoc in the same time.
I cannot make a suggestion here about non modified code, but it seems legit to me to update that.
```suggestion
     * @param string $hostList
     * @param CentreonXMLBGRequest $objXMLBG
     * @param string $o
     * @param false|int $instance
     * @param false|int $hostgroups
     */
```

Reply:
I was really lazy to do it :)